impeller/renderer/backend/gles/reactor_gles.h

```diff
@@ -231,11 +249,18 @@ class ReactorGLES {
     std::optional<GLuint> name;
     std::optional<std::string> pending_debug_label;
     bool pending_collection = false;
+    fml::closure callback = {};
```

Comment:
fml::ScopedCleanupClosure so you don't need to remember to invoke it.

Reply:
Done